survey/templates/survey/survey.html

```diff
@@ -27,7 +27,7 @@ <h3 class="panel-title"> <a
 			            </a></h3>
 				    </div>
                     <div class="category-container">
-				        <div id="collapse{{category.slugify}}" class="panel-collapse collapse
+				        <div id="collapse{{category.slugify}}" class="panel-collapse collapse {% if not survey.editable_answers and response_form.response is not None %}in{% endif %}
```

Comment:
Does it mean the content will be hidden if the user can't edit? Wouldn't it be better to display a page saying that the user already answered?

Reply:
This means that when the user has already answered _and_ the survey is not editable, all answers are directly displayed (whereas, when answering or editing, the categories are hidden, by default).